04/homework.py

```diff
@@ -0,0 +1,12 @@
+message = input('Please write something: ')
+if message.isdigit():
+    print('This is an integer number')
+    if int(message) % 2 == 0:
+        print(f'Number {message} is even')
+    else:
+        print(f'Number {message} is odd')
+elif message.isalpha():
```

Comment:
This check can lead to unpredictable behavior when words with apostrophes will be entered.
It is not a big case for English, but other languages can contain many such words.

Within this task we can assume that everything that is not a digit is a word